### PR TITLE
helm: handle multiline output

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/helm.py
plugins/modules/helm_plugin_info.py

### CODE REVIEW

File: plugins/modules/helm_plugin.py [hunk 1]
```diff
@@ -198,6 +198,8 @@ def main():
                     continue
                 name, dummy, dummy = line.split('\t', 3)
                 name = name.strip()
+                if name == "":
```
Comment (abikouo) [inline plugins/modules/helm_plugin.py:201]: what if we put this piece of code which is common to the 3 modules into a function in module_utils/helm.py ?

Comment (Akasurde) [reply plugins/modules/helm_plugin.py:201]: Yes, we can. Thanks,

**Comment by gravesm** [comment]
@Akasurde could you re-open this in kubernetes.core? Also, would be good to address @abikouo's comment, then we can get this merged.

**Comment by Akasurde** [comment]
superseded by https://github.com/ansible-collections/kubernetes.core/pull/64

**Pull request closed without merging**